packages/tinacms/src/unifiedClient/index.ts

```diff
@@ -102,6 +117,18 @@ export class TinaClient<GenQueries> {
       ...providedFetchOptions,
     };
 
+    //? Look for the header and change to use this branch instead of the build time generated branch.
+    //? This comes from the clients fetch options:
```

Comment:
```suggestion
    // Look for the header and change to use this branch instead of the build time generated branch.
    // This comes from the clients fetch options:
```